internal/app/singularity/cache_clean_linux.go

```diff
@@ -51,6 +66,11 @@ func cleanBlobCache() error {
 
 // CleanCache : clean a type of cache (cacheType string). will return a error if one occurs.
```

Comment:
Can you fix this silly godoc comment thing? I know it was me that wrote this a while ago...

```suggestion
// CleanCache will clean a type of cache (cacheType). The possible options are: 'library', 'oci', 'blob', and 'all'.
Will return a err if one occurs.
```

Reply:
done